pkg/providers/amifamily/ami_test.go

```diff
@@ -304,10 +304,7 @@ var _ = Describe("AMIProvider", func() {
 							Values: aws.StringSlice([]string{"my-ami"}),
 						},
 					},
-					Owners: []string{
-						"amazon",
-						"self",
-					},
+					Owners: []string{},
```

Comment:
nit : Probably can change the name of the test to be more deliberate that we're expecting "empty owners" rather than inferring that the default is empty